back/lib/participation_method/native_survey.rb

```diff
@@ -18,6 +18,11 @@ def assign_defaults(input)
 
     def create_default_form!
       form = CustomForm.create(participation_context: participation_context)
+      CustomField.create(
+        resource: form,
+        input_type: 'page',
+        title_multiloc: MultilocService.new.i18n_to_multiloc('form_builder.default_question_page.title')
```

Comment:
Perhaps we could consider a solution where the title remains up to date with the default copy until this has been manually updated? This could be realized e.g. by setting the `title_multiloc` to `nil` initially and letting the model return the latest translations when getting `title_multiloc` in that case. We did something similar for the nav bar items, although this approach has undergone some changes.

Reply:
This is not an issue anymore, because it has been decided that the default question page should not have a title. I removed the default title. Given that the `title_multiloc` of a field is required for all fields except page fields, I had to adapt the validation of `title_multiloc`.